### PR TITLE
updated accordion, list widget, long tap and removed duplicate

### PR DESCRIPTION
**Pull request by RitupL**

### CODE REVIEW

File: learn/app-development/widgets/basic/anchor.md [hunk 1]
```diff
@@ -88,4 +88,5 @@ From the Style tab of Properties panel,  you can set the Anchor style. You can
 | **Touch Events** |
 | On Tap | This event handler is called whenever the widget is tapped. |
 | On Double Tap | This event handler is called whenever the widget is double tapped. |
+| On long tap | This event handler is called whenever the long tap event is triggered on a widget. |
```
Comment (sboyina) [inline learn/app-development/widgets/basic/anchor.md:91]: Long Tap event is present only in React Native apps.

File: learn/app-development/widgets/basic/label.md [hunk 2]
```diff
@@ -56,4 +56,5 @@ The following features for Label can be set in the **Class** property from the *
 | **Touch Events** |
 | On tap | This event handler is called whenever the tap event is triggered on a widget. |
 | On double tap | This event handler is called whenever the double tap event is triggered on a widget. |
+| On long tap | This event handler is called whenever the long tap event is triggered on a widget. |
```
Comment (sboyina) [inline learn/app-development/widgets/basic/label.md:59]: Long Tap event is present only in React Native apps.

File: learn/app-development/widgets/basic/picture.md [hunk 3]
```diff
@@ -63,3 +63,4 @@ You can add customizations from the **Properties** panel such as **Width** and *
 | **Touch Events** |
 | On tap | This event handler is called whenever the tap event is triggered on a widget. |
 | On double tap | This event handler is called whenever the double tap event is triggered on a widget. |
+| On long tap | This event handler is called whenever the long tap event is triggered on a widget. |
```
Comment (sboyina) [inline learn/app-development/widgets/basic/picture.md:66]: Long Tap event is present only in React Native apps.

File: learn/app-development/widgets/basic/progress-bar.md [hunk 4]
```diff
@@ -46,6 +46,7 @@ id: "progress-bar"
 | **Touch Events** |
 | On tap | This event handler is called whenever the tab event is triggered on a widget. |
 | On double tap | This event handler is called whenever the double tap event is triggered on a widget. |
+| On long tap | This event handler is called whenever the long tap event is triggered on a widget. |
```
Comment (sboyina) [inline learn/app-development/widgets/basic/progress-bar.md:49]: Long Tap event is present only in React Native apps.

File: learn/app-development/widgets/basic/progress-circle.md [hunk 5]
```diff
@@ -39,6 +39,7 @@ id: "progress-circle"
 | **Touch Events** |
 | On-tap | This event handler is called whenever the tab event is triggered on a widget. |
 | On-double tap | This event handler is called whenever the double tap event is triggered on a widget. |
+| On-long tap | This event handler is called whenever the long tap event is triggered on a widget. |
```
Comment (sboyina) [inline learn/app-development/widgets/basic/progress-circle.md:42]: Long Tap event is present only in React Native apps.

File: learn/app-development/widgets/container/container.md [hunk 6]
```diff
@@ -41,6 +41,7 @@ You can also use the Conditional Class property from the Style tab to set the cl
 | **Touch Events** |
 | On tap | This event handler is called whenever the tap event is triggered on a widget. |
 | On double tap | This event handler is called whenever the double tap event is triggered on a widget. |
+| On long tap | This event handler is called whenever the long tap event is triggered on a widget. |
```
Comment (sboyina) [inline learn/app-development/widgets/container/container.md:44]: Long Tap event is present only in React Native apps.

File: learn/app-development/widgets/container/tile.md [hunk 7]
```diff
@@ -44,6 +44,7 @@ You can also use the Conditional Class property from the Style tab to set the cl
 | **Touch Events** |
 | On tap | This event handler is called whenever the tap event is triggered on a widget. |
 | On double tap | This event handler is called whenever the double tap event is triggered on a widget. |
+| On long tap | This event handler is called whenever the long tap event is triggered on a widget. |
```
Comment (sboyina) [inline learn/app-development/widgets/container/tile.md:47]: Long Tap event is present only in React Native apps.

File: learn/app-development/widgets/form-widgets/button.md [hunk 8]
```diff
@@ -94,6 +94,7 @@ From the Style tab of Properties panel, you can also set the button size along w
 | **Touch Events** |
 | On tap | This event handler is called whenever the widget is tapped. |
 | On double tap | This event handler is called whenever the widget is double tapped. |
+| On long tap | This event handler is called whenever the long tap event is triggered on a widget. |
```
Comment (sboyina) [inline learn/app-development/widgets/form-widgets/button.md:97]: Long Tap event is present only in React Native apps.

**Review by sboyina — APPROVED** [review]

**Comment by swethakundaram** [comment]
Update URLs

**Review by swethakundaram — APPROVED** [review]

**Pull request merged**